app/src/main/java/org/connectbot/ConsoleActivity.java

```diff
@@ -563,17 +564,21 @@ public boolean onTouch(View v, MotionEvent event) {
 						MotionEventCompat.getSource(event) == InputDevice.SOURCE_MOUSE &&
 						event.getAction() == MotionEvent.ACTION_DOWN) {
 					switch (event.getButtonState()) {
-					case MotionEvent.BUTTON_PRIMARY:
-						// Automatically start copy mode if using a mouse.
-						startCopyMode();
-						break;
+						case MotionEvent.BUTTON_PRIMARY:
+							// Automatically start copy mode if using a mouse.
+							startCopyMode();
```

Comment:
Not sure this still makes sense... Do you have a bluetooth mouse you can test with? If not, I can take a look into mouse functionality after this lands.